CorsixTH/Lua/hospitals/player_hospital.lua

```diff
@@ -48,6 +48,20 @@ end
 
 --! Give advice to the player at the end of a day.
 function PlayerHospital:dailyAdviceChecks()
+  --[[
+    Current checks summarised as below
+    Day 1: Money advice
+    Day 3: Staff rooms
+    Day 5: Radiators
+    Day 8: Toilets
+    Day 10: Long queues
+    Day 12: Benches
+    Day 15: Heating (patients)
+    Day 18: Plants (No handymen)
+    Day 20: Heating (staff) NB: only patient or staff executes per month (if any)
+    Day 24: Drinks machines
+  --]]
```

Comment:
Busy little guy that advisor eh?

Did we ensure the safety of the advisor sufficiently against angry clueless users that got mad due to all its messages spam?

Reply:
We do not, but it seems if you play the game right they shouldn't all trigger.
You could argue some could be set to execute a % of the time using the `rand_frac` parameter on `giveAdvice`.

I'm not sure how consistent our amount of announcements compares to TH.